examples/fetch_content/src/main.cpp

```diff
@@ -0,0 +1,46 @@
+#include <sqlite_orm/sqlite_orm.h>
+
+#include <string>
+#include <iostream>
+#include <cassert>
+
+using namespace sqlite_orm;
```

Comment:
what is this test created for? What does it mean `fetch_content`? If it is one of the examples why is has its own directory instead of a single file with a custom name just like all other examples?

Reply:
I found the answer in the first message.